apps/prairielearn/src/sprocs/authz_assessment.sql

```diff
@@ -21,6 +21,7 @@ CREATE FUNCTION
 AS $$
 DECLARE
     user_result record;
+    
```

Comment:
I see a number of random newlines in this diff; is there a reason for them?

Reply:
I added some code and forgot to remove the whitespace after deleting it, I will do it now.